meta-balena-common/classes/image_types_balena.bbclass

```diff
@@ -1,4 +1,6 @@
-inherit image_types balena-engine-rootless
+inherit image_types
+ENGINE_CLIENT ?= "docker"
+inherit ${@bb.utils.contains('BALENA_STORAGE','overlay2','balena-engine-rootless', '', d)}
```

Comment:
Are we planning to remove these conditions once aufs is deprecated or should they be squashed with the original inherit changes?

Reply:
Yes, we'll remove them as soon as aufs is deprecated.